topics/variant-analysis/tutorials/variant-calling/tutorial.md

```diff
@@ -0,0 +1,605 @@
+---
+layout: tutorial_hands_on
+title: "Variant Calling Workflow"
+questions:
+  - "How do I find sequence variants between my sample and a reference genome?"
+objectives:
+  - "Understand the steps involved in variant calling."
+  - "Describe the types of data formats encountered during variant calling."
+  - "Use command line tools to perform variant calling."
+time_estimation: "50 min"

```

Comment:
```suggestion
time_estimation: "50M"

```
(we use a standardized format for time indications)

Reply:
Thank you, I changed it to the proper format.